tools/psa/release.py

```diff
@@ -74,7 +74,14 @@ def _get_target_info(target):
     if not os.path.exists(delivery_dir):
         raise Exception("{} does not have delivery_dir".format(target))
 
-    return tuple([TARGET_MAP[target].name,
+    if toolchain:
+        if not TOOLCHAIN_CLASSES[toolchain].check_executable():
```

Comment:
```suggestion
        if toolchain not in TARGET_MAP[target].supported_toolchains:
```

Reply:
@jeromecoutant it should be like this
the executeable_check is done in main()

Reply:
done